app/models/vacancy.rb

```diff
@@ -123,13 +123,27 @@ class Vacancy < ApplicationRecord
 
   acts_as_gov_uk_date :starts_on, :ends_on, :publish_on, :expires_on
 
-  scope :applicable, (-> { where('expires_on >= ?', Time.zone.today) })
+  scope :applicable, (-> { applicable_by_date.or(applicable_by_time) })
+  scope :applicable_by_time, (-> { where.not(expiry_time: nil).where('expiry_time >= ?', Time.zone.now) })
```

Comment:
I could be wrong but could we condense the SQL queries to use only one where statement?

Reply:
Possibly, although the first `where` is a `where.not` 🤔

Reply:
We could probably have something like `where(condition AND NOT condition)`. 
There are also other places where we're doing `.where(condition).where(condition)`. 
Could these be written as `where(condition AND condition)`?

Reply:
Resolved in 2a700b8